### PR TITLE
The frontend and dashboard should not be removed to core

### PR DESCRIPTION
**Pull request by luprz**
ref #587 

The developer, when entering the rocket administration, should not be able to uninstall the interface and dashboard rockets, so as not to damage the structure of the core.

Solution:
- Block the uninstall button in the view and action.

### CODE REVIEW

**Comment by anyelopetit** [comment]
@luprz What should I check to make this PR?

**Comment by luprz** [comment]
@anyelopetit If it's done in a good way

File: app/policies/rocket_policy.rb [hunk 1]
```diff
@@ -2,10 +2,30 @@
...
+  def create?
+    keppler_admin?
+  end
+
+  def install?
+    keppler_admin?
+  end
+
+  def uninstall?
+    @object.name && !Rocket.core_depending.include?(@object.name)
```
Comment (pdrowr) [inline app/policies/rocket_policy.rb]: Aquí puedes cambiar ésto:

`!Rocket.core_depending.include?(@object.name)`

Por ésto: 

`Rocket.core_depending.exclude?(@object.name)`

Comment (luprz) [reply app/policies/rocket_policy.rb]: Nice!

**Pull request merged**